src/Pages/LoginPage/index.js

```diff
@@ -60,7 +60,7 @@ export default function LoginPage() {
                   <Icon name="bananaIcon" size={200} />
                 </Col>
                 <Col sm={6}>
-                  <h1 className={styles.title}>BANANA PORTAL</h1>
+                  <h1 className={styles.title}>BRENDAN PORTAL 3</h1>
```

Comment:
Once you switch this back to `BANANA PORTAL`, you can go ahead and merge it!

Reply:
@brendobrendo @tackc yep I agree, looks good 🚀